addons/api/addon/models/target.js

```diff
@@ -31,6 +31,16 @@ export default class TargetModel extends GeneratedTargetModel {
   })
   brokered_credential_source_ids;
 
+  /**
+   * Injected Application Credential source ids are read only and can be
+   * persisted via a dedicated call to `addInjectedApplicationCredentialSources()`.
+   */
+  @fragmentArray('fragment-string', {
```

Comment:
Does this depend on randy's PR that changed these `fragmentArray`s?

Reply:
Yes, this part of the code has been changed with that pr. I'll pull from the Main when those changes are merged